packages/iov-bns/src/bnsconnection.spec.ts

```diff
@@ -1907,4 +1908,85 @@ describe("BnsConnection", () => {
 
     connection.disconnect();
   });
+
+  describe("getFeeQuote", () => {
+    it("works for send transaction", async () => {
+      pendingWithoutBnsd();
+      const connection = await BnsConnection.establish(bnsdTendermintUrl);
```

Comment:
Might want to consider putting this connection stuff in a `beforeEach`/`afterEach` hook since it's used so often.

Reply:
Yeah, right, this is a little annoying. In our experience, pending() does not work properly with beforeEach(). Also this adds some top level mutable state, which is not great as well.

Reply:
I've pretty much accepted top level mutable state as a necessary evil for concise mocha tests and it seems the same in jasmine. The pending/beforeEach conflict's annoying though.

Reply:
The `beforeEach` hook shouldn't run for pending tests though, right? In that case we could at least move the connection stuff out of the test.

Another slightly weirder thought: conditionally call `it()` or `it.skip()` depending on whether the test should be executed. (Probably involves a lot of ugly bracketing.)

Reply:
pendingWithoutBnsd can only know at runtime if the test is pending. So beforeEach is called in any case. However, a [call to pending in beforeEach does not stop the test execution](https://github.com/jasmine/jasmine/issues/1579).

Using `xit` instead of `it` does not allow to secify a pending reason: https://jasmine.github.io/api/edge/global.html#xit